external/storm-kafka/src/jvm/storm/kafka/PartitionManager.java

```diff
@@ -149,9 +150,9 @@ public EmitState next(SpoutOutputCollector collector) {
             }
             
             if ((tups != null) && tups.iterator().hasNext()) {
-                if(_spoutConfig.topicAsStreamId) {
+               if (!Strings.isNullOrEmpty(_spoutConfig.outputStreamId)) {
```

Comment:
Isn't it an overkill to import a whole package (com.google.common.base.Strings) to only test for null or empty? Can't we just use: 
if (_spoutConfig.outputStreamId!=null && _spoutConfig.outputStreamId!="") 
or something to that effect instead of importing a new library?